src/app/_components/ClickTooltip.tsx

```diff
@@ -0,0 +1,64 @@
+'use client'
+
+import { useId, useState, type ReactNode } from 'react'
+
+import * as RadixPopover from '@radix-ui/react-popover'
+import { clsx } from 'clsx'
+
+type TooltipChildrenRenderProps = {
+  open: boolean
+  tooltipId: string
+}
+
+export type ClickTooltipProps = {
+  children: ReactNode | ((props: TooltipChildrenRenderProps) => ReactNode)
+  description: string
+  gap?: RadixPopover.PopoverContentProps['sideOffset']
+  side?: RadixPopover.PopoverContentProps['side']
+}
+
+const animationClasses = {
+  bottom: 'data-[state=open]:data-[side=bottom]:animate-slideUpAndFade',
+  left: 'data-[state=open]:data-[side=left]:animate-slideRightAndFade',
+  right: 'data-[state=open]:data-[side=right]:animate-slideLeftAndFade',
+  top: 'data-[state=open]:data-[side=top]:animate-slideDownAndFade',
+}
+
+export function ClickTooltip({
```

Comment:
What about `PopoverTooltip`?

Reply:
I find `ClickTooltip` more descriptive.

If we have `Tooltip` and `PopoverTooltip`, do you know which one opens on click vs on hover?

But that is, of course, if we end up building two tooltip components.

Reply:
Let's call it `Tooltip`